docs/troubleshooting/parameter-sweeping-errors.md

```
@@ -0,0 +1,317 @@
+# Parameter Sweeping Error Troubleshooting Guide
+
+This guide helps you resolve common errors when using AIPerf's parameter sweeping feature.
+
+## Common Errors and Solutions
+
+### 1. Invalid Concurrency Value
+
+**Error Message:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)